src/pocketmine/block/Grass.php

```diff
@@ -63,16 +63,20 @@ public function getDrops(Item $item){
 
 	public function onUpdate($type){
 		if($type === Level::BLOCK_UPDATE_RANDOM){
-			//TODO: light levels
-			$x = mt_rand($this->x - 1, $this->x + 1);
-			$y = mt_rand($this->y - 2, $this->y + 2);
-			$z = mt_rand($this->z - 1, $this->z + 1);
-			$block = $this->getLevel()->getBlock(new Vector3($x, $y, $z));
```

Comment:
I could understand that @PEMapModder didn't realize the getBlock leak problem since @shoghicp added it, but `new Vector3($this->x, $this->y, $this->z)` was totally pointless...